pwnlib/elf/elf.py

```diff
@@ -852,6 +852,12 @@ def _patch_elf_and_read_maps(self):
             log.warn_once("Injected /proc/self/maps code did not execute correctly")
             return {}
 
+        # Sometimes the original binary already fail to run, for example,
+        # in case glibc mismatches.
+        if 'not found (required by' in data:
+            log.warn_once(f"Cannot execute `{self.path}` to get /proc/self/maps")
```

Comment:
```suggestion
            log.warn_once("Cannot execute `%s` to get /proc/self/maps", self.path)
```

Will also avoid errors if path happens to contain %

Reply:
I've tested f-string with a var containing %. Nothing wrong with the output string. But I'll commit your change to make it more uniform with the rest of the file.

Reply:
Sure, but warn_once does %-formatting after the f-string gets interpolated:
```py
>>> from pwn import*
>>> log.warn_once('%s', 'abc')
[!] abc
>>> log.warn_once(f'%s')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/arusekk/src/my-git/pwntools/pwnlib/log.py", line 380, in warn_once
    return self.warning_once(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/arusekk/src/my-git/pwntools/pwnlib/log.py", line 372, in warning_once
    m = message % args
        ~~~~~~~~^~~~~~
TypeError: not enough arguments for format string
```